src/go/privileged-service/README.md

```diff
@@ -26,4 +26,9 @@ To run the Rancher Desktop Privileged Service you can use the following availabl
 
 ```pwsh
 .\privileged-service.exe install | start | stop | uninstall
-```
\ No newline at end of file
+```
+
+Note: the `install` command must be run as an administrator.
```

Comment:
The `uninstall` command needs to be run as admin (to match) too.

Reply:
Updated!